hyperspy/_components/lorentzian.py

```diff
@@ -16,76 +16,78 @@
 # You should have received a copy of the GNU General Public License
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
-import numpy as np
+from hyperspy._components.expression import Expression
 
-from hyperspy.component import Component
+class Lorentzian2(Expression):
 
-
-class Lorentzian(Component):
-
-    r"""Cauchy-Lorentz distribution (a.k.a. Lorentzian function) component
+    r"""Cauchy-Lorentz distribution (a.k.a. Lorentzian function) component.
 
     .. math::
 
         f(x)=\frac{a}{\pi}\left[\frac{\gamma}{\left(x-x_{0}\right)^{2}+\gamma^{2}}\right]
 
-    +---------------------+-----------+
-    |     Parameter       | Attribute |
-    +---------------------+-----------+
-    +---------------------+-----------+
-    |      :math:`a`      |     A     |
-    +---------------------+-----------+
-    |    :math:`\gamma`   |   gamma   |
-    +---------------------+-----------+
-    |      :math:`x_0`    |  centre   |
-    +---------------------+-----------+
+    ================= ===========
+      Parameter        Attribute 
+    ================= ===========
+      :math:`a`         A     
+      :math:`\gamma`   gamma   
+      :math:`x_0`      centre   
+    ================= ===========
+    
+    Parameters
+    -----------
+        A : float
+            Height parameter, where :math:`A/\gamma` is the maximum of the peak.
+        gamma : float
+            Scale parameter corresponding to the half-width-at-half-maximum of the peak, which corresponds to the interquartile spread.
+        centre : float
+            Location of the peak maximum.
+        **kwargs
+            Extra keyword arguments are passed to the ``Expression`` component.
+            An useful keyword argument that can be used to speed up the
+            component is `module`. See the ``Expression`` component
```

Comment:
I would remove this sentence: "An useful keyword argument that can be used to speed up the component is `module`." because `numexpr` is now already the default, there is currently no room left for speed improvement...
Until recently, `numexpr` was an optional dependency and `numpy` was used by default, so I guess this sentence was mentioned in a few places so that people are informed and they can install and use `numexpr` if there can/wish.